bsoid_app/bsoid_utilities/load_workspace.py

```diff
@@ -2,6 +2,8 @@
 import os
 import joblib
 
+working_dir = os.environ.get('WORKING_DIR_BSOID', '')
```

Comment:
I don't think you need these here because you no longer call `query_workspace` in the main app